iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -532,7 +542,7 @@ struct SerializeParallelLoopPattern
 template <typename LinalgOpTy>
 static LogicalResult mapLinalgOpToLocalInvocationIdImpl(
     LinalgOpTy linalgOp, ArrayRef<Value> operands,
-    ConversionPatternRewriter &rewriter) {
+    ConversionPatternRewriter &rewriter, bool optimizeControlFlow) {
```

Comment:
Do we need this at all? I thought we didn't want to do this anymore? Does it do anything right now?

Reply:
mostly for legacy code. I didnt clean it up, cause this whole pass is probably going to go away.